altiProfilAdmin/controllers/config.classic.php

```diff
@@ -24,8 +24,13 @@ class configCtrl extends jController {
 
     function __construct( $request ) {
         parent::__construct( $request );
-        $monfichier = jApp::configPath('altiProfil.ini.php');
-        $this->ini = new jIniFileModifier ($monfichier);
+        if (method_exists('jApp', 'varConfigPath')) {
+		    $monfichier = \jApp::varConfigPath('altiProfil.ini.php');
+            $this->ini = new \Jelix\IniFile\IniModifier($monfichier);
+        } else {
+		    $monfichier = \jApp::configPath('altiProfil.ini.php');
+            $this->ini = new jIniFileModifier($monfichier);
```

Comment:
```suggestion
        if (method_exists('jApp', 'varConfigPath')) {
	    // LWC >= 3.7
	    $monfichier = \jApp::varConfigPath('altiProfil.ini.php');
            $this->ini = new \Jelix\IniFile\IniModifier($monfichier);
        } else {
	    $monfichier = \jApp::configPath('altiProfil.ini.php');
            $this->ini = new jIniFileModifier($monfichier);
```

Mixed tab ?

Reply:
My IDE drives me crazy, unable to get a correct indentation without modifiying the whole file i gave up 😭 😄